sync_diff_inspector/config/config.go

```diff
@@ -180,12 +180,8 @@ func (t *TaskConfig) Init(
 		return errors.Trace(err)
 	}
 
-	// Set default value when output is empty
 	if t.OutputDir == "" {
-		t.OutputDir = timestampOutputDir()
-		if err := os.RemoveAll(t.OutputDir); err != nil && !os.IsNotExist(err) {
-			log.Fatal("fail to remove the temp directory", zap.String("path", t.OutputDir), zap.String("error", err.Error()))
-		}
+		return errors.New("output-dir is missing from the task configuration")
```

Comment:
🤔 seems we will generate a temporary output-dir

Reply:
PTAL again